app/models/game.rb

```diff
@@ -14,6 +14,30 @@ def populate!
     populate_right_white_half!
   end
 
+  def white_check?
```

Comment:
I was getting method is too long (11/10) lines. Didn't see the way to make it work in one, so split it into two.